CHANGES.md

```diff
@@ -1,6 +1,427 @@
+  (see https://chapel-lang.org/docs/1.26/modules/standard/BigInteger.html#BigInteger.primality  
+  and https://chapel-lang.org/docs/1.26/modules/standard/BigInteger.html#BigInteger.bigint.probablyPrime)
+* swapped the order of the values returned from `bigint.getD2Exp()`  
+  (see https://chapel-lang.org/docs/1.26/modules/standard/BigInteger.html#BigInteger.bigint.getD2Exp)
+
+Package Modules
+---------------
+* added a new 'CopyAggregation' module supporting aggregated copies  
+  (see https://chapel-lang.org/docs/1.26/modules/packages/CopyAggregation.html)
+* added a new 'Socket' package module to support network programming  
+  (see https://chapel-lang.org/docs/1.26/modules/packages/Socket.html)
+* added a new 'Channel' package module to support communicating between tasks  
+  (see https://chapel-lang.org/docs/1.26/modules/packages/Channel.html)
+* changed 'TOML' package to sort keys when writing them out  
+  (see https://chapel-lang.org/docs/1.26/modules/packages/TOML.html)
+
+Standard Domain Maps (Layouts and Distributions)
+------------------------------------------------
+
+Tool Improvements
+-----------------
+* added a script for displaying available runtime configurations  
+  (see `$CHPL_HOME/util/chplenv/printchplbuilds.py`)
+
+Performance Optimizations / Improvements
+----------------------------------------
+* significantly improved fine-grained communication on Slingshot-11 networks
+* made promotions with slices more efficient with `-schpl_serializeSlices=true`
+
+Compilation-Time / Generated Code Improvements
+----------------------------------------------
+
+Memory Improvements
+-------------------
+* eliminated a minor amount of memory allocated/freed when iterating over types
+* fixed a memory leak for `set.add()` in the 'Set' module
+
+Documentation
+-------------
+* added documentation to the language specification for the `manage` statement  
+  (see https://chapel-lang.org/docs/1.26/language/spec/statements.html#the-manage-statement)
+* merged domain/array 'built-in types and functions' content into language spec  
+  (see https://chapel-lang.org/docs/1.26/language/spec/domains.html#module-ChapelDomain  
+   and https://chapel-lang.org/docs/1.26/language/spec/arrays.html#module-ChapelArray)
+* updated GPU technote to expand on currently known limitations  
+  (see https://chapel-lang.org/docs/1.26/technotes/gpu.html)
+* added documentation for many methods in the 'BigInteger' module  
+  (see https://chapel-lang.org/docs/1.26/modules/standard/BigInteger.html)
+* stopped documenting the existence of the `bigint.mpz` field  
+  (see https://chapel-lang.org/docs/1.26/modules/standard/BigInteger.html#BigInteger.bigint)
+* added documentation for which elements are preserved in `set` operations  
+  (see https://chapel-lang.org/docs/1.26/modules/standard/Set.html)
+
+Syntax Highlighting
+-------------------
+
+Example Codes
+-------------
+* removed 'SSCA2' from the `test/release/examples/benchmarks` directory
```

Comment:
Maybe point to the PR on this one so that some of the reasoning for the removal is easily accessible?

Reply:
The PR is #19054, the information is in the "details" section

Reply:
I'd be okay with including a link to it, but also okay with not doing so, figured I should at least make it accessible for making the decision

Reply:
I ended up just putting the rationale into the entry itself since the `test/release/` part of the benchmark isn't user-facing and `examples/` is implied in the section name.  I also added a similar rationale for the bullet after this one.  So:

```
Example Codes
-------------
* removed 'benchmarks/ssca2' due to its	use of old styles and lack of upkeep
* removed `patterns/recordio.chpl` due to its use of `iostyle`
```